test/test_unit/validation-errors-converter.test.js

```diff
@@ -3,163 +3,226 @@ const SchemaValidator = require('../../src/schema-validator')
-                format: 'httpUrl'
-            },
-            formatError: {
-                type: 'string',
-                format: 'email'
             }
         }
-    }
-    const schemaValidator = new SchemaValidator(schema.$id, [schema])
+        const schemaValidator = new SchemaValidator(schema.$id, [schema])
 
-    it('type', () => {
         const data = {
             typeError: {}
         }
 
-        const result = schemaValidator.validate(data)
-        expect(result).to.include({
+        const result = schemaValidator.validate.bind(schemaValidator, data)
+        expect(result).to.throw().to.include({
             field: '.typeError',
             message: "Property 'typeError' must be string.",
             rejected_value: data.typeError
         })
     })
 
     it('additionalProperties', () => {
+        const schema = {
+            $id: 'http://example.com/schemas/schema.json',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                additionalPropertyError: {
+                    type: 'object',
+                    additionalProperties: false,
+                    properties: {
+                        a: {
+                            type: 'string'
+                        }
+                    }
+                }
+            }
+        }
+        const schemaValidator = new SchemaValidator(schema.$id, [schema])
+
         const data = {
             additionalPropertyError: {
                 d: {}
             }
         }
 
-        const result = schemaValidator.validate(data)
-        expect(result).to.include({
+        const result = schemaValidator.validate.bind(schemaValidator, data)
+        expect(result).to.throw().to.include({
             field: '.additionalPropertyError.d',
             message: "Property 'd' is not supported."
         })
     })
 
     it('required', () => {
+        const schema = {
+            $id: 'http://example.com/schemas/schema.json',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                requiredError: {
+                    type: 'object',
+                    required: ['a'],
+                    properties: {
+                        a: {
+                            type: 'string'
+                        }
+                    }
+                }
+            }
+        }
+        const schemaValidator = new SchemaValidator(schema.$id, [schema])
+
         const data = {
             requiredError: {}
         }
 
-        const result = schemaValidator.validate(data)
-        expect(result).to.include({
+        const result = schemaValidator.validate.bind(schemaValidator, data)
+        expect(result).to.throw().to.include({
             field: '.requiredError',
             message: "Missing property 'a'."
         })
     })
 
     it('const', () => {
+        const schema = {
+            $id: 'http://example.com/schemas/schema.json',
```

Comment:
It seems that all the schemas have the same base:
```
{
            $id: 'http://example.com/schemas/schema.json',
            type: 'object',
            additionalProperties: false
}
```

I hope that you already know how to dry them. 😉